src/OAuth/ITokenClient.cs

```diff
@@ -24,9 +25,10 @@ public partial interface ITokenClient
         /// <param name="redirectUri">Authorization endpoint redirect URI.</param>
         /// <param name="clientId">OAuth application client ID.</param>
         /// <param name="clientSecret">OPTIONAL OAuth application client secret. Required for web apps.</param>
+        /// <param name="scope">(optional) The requested space-delimited scopes for the access token.</param>
         /// <param name="codeVerifier">OPTIONAL PKCE code verifier. Required for SPA apps.</param>
         /// <returns>A response that contains an access token.</returns>
-        Task<GetAccessTokenResponse> GetAccessTokenFromCode(string code, string redirectUri, string clientId, string clientSecret = null, string codeVerifier = null);
+        Task<GetAccessTokenResponse> GetAccessTokenFromCode(string code, string redirectUri, string clientId, string clientSecret = null, string scope = null, string codeVerifier = null);
```

Comment:
should the scope be the last parameter? It is the last parameter in the implementation

Reply:
Right, fixed.